tabletop/core/view.js

```diff
@@ -275,9 +276,100 @@ View.prototype.drawMessage = function() {
 
 View.prototype.animate = function() {
   this.updateTokens();
+  this.updateAllPlayersInfo();
   requestAnimationFrame(this.animate.bind(this));
   this.renderer.render(this.stage);
 };
 
+View.prototype.drawAllPlayersInfo = function() {
+
+  var playersInfo = this.getPlayersInfo();
+  if (!playersInfo) return;
+
+  var infoBlock = new PIXI.Graphics();
+  infoBlock.x = c.boardWidth + c.leftBuffer * 2;
+  infoBlock.y = c.upperBuffer;
+
+  var blockSize = 150;
+  var vertOffset = .2*c.upperBuffer;
+  
+  var view = this;
+  playersInfo.forEach(function(player) { 
+    view.drawPlayerInfo(player, vertOffset, infoBlock);
+    vertOffset += blockSize;
+  });
+  
+  this.stage.addChild(infoBlock);
+  return infoBlock;
+};
+
+
+View.prototype.drawPlayerInfo = function(player, vertOffset, infoBlock) {
+  
+  var string = "";
+  if (player["name"]) string += player["name"] + "\n";
+  for (var key in player) { 
+    if (key == "name") continue; 
+    string += key + ":  " + player[key] + "\n";
+  } 
+  
+  var info = new PIXI.Text(string, 
+                           {font: '20px Arial',
+                            align : 'left',
+                            wordWrap : true,
+                            strokeThickness : .25,
+                            wordWrapWidth : c.canvasWidth - c.boardWidth - (4 * c.leftBuffer)
+                           });
+
+  info.x = c.leftBuffer * 0.2;
+  info.y = vertOffset;
+  
+  var box = new PIXI.Graphics();
+  box.y = vertOffset - c.upperBuffer*.2;
+  
+  var outline = new PIXI.Graphics();
+  outline.y = vertOffset - c.upperBuffer*.2;
+  outline.lineStyle(1, 0, 1);
+  outline.drawRect(0, 0, c.canvasWidth - c.boardWidth - (3 * c.leftBuffer), 140);
+  
+  box.lineStyle(1, 0, 1);
+  box.beginFill(0x44C0DF, 1);
+  box.drawRect(0, 0, c.canvasWidth - c.boardWidth - (3 * c.leftBuffer), 140);
+  
+  infoBlock.addChild(outline);
+  infoBlock.addChild(box);
+  infoBlock.addChild(info);
+};
+
+
+/* 
+ should be overridden in subclass to display player info panel 
+ "name" field displays first without label
+ info should be an array of dictionaries, ie: 
+
+  return [    
+    { 
+      name: "kc", 
+      money: 234, 
+      skill: 10
+    },
+    
+    { 
+      name: "john",
+      money: 98, 
+      skill: 1
+    }
+  ];
+
+*/
```

Comment:
Can you look at all the other methods' comments and use the jsdoc syntax so we can build the docs?